### ISSUE TITLE
train

### ISSUE THREAD

**Issue by yaningjm** [gy65896/SCANet#4]
Why is the following error always reported when training the NTIRE2023 data set: Out of memory

**Comment by YuanGao-YG** [comment]
This error may be caused by insufficient GPU memory. Please try using a GPU with larger memory or reducing the size of the training image.

**Comment by yaningjm** [comment]
Thank you for your reply. The experiment using NVIDIA RTX A6000 50G video memory will also report an error of exceeding the memory. I guess there is a problem with the framed part in the picture, because the NTIRE2023 data set is too large and the test part cannot be loaded. Have you encountered such a problem during training? If so, can you tell me how to solve it?
![SCANet_1](https://github.com/gy65896/SCANet/assets/116944719/f5cfa4a1-36ae-44fb-a9cd-205ee94e6f43)
@YuanGao-YG

**Comment by gy65896** [comment]
You are right. During testing, we use a scaled-down version for testing. If you want to test on the original size, you need A100 80G for testing.

**Comment by yaningjm** [comment]
Ok, I see. Thank you very much
